src/Sulu/Bundle/PageBundle/Content/Types/Date.php

```diff
@@ -20,6 +20,8 @@
  */
 class Date extends SimpleContentType
 {
+    const FORMAT = 'Y-m-d\TH:i:s';
```

Comment:
Why did that format change? This seems like the format for the `DateTime`, doesn't it?

Reply:
you are right, thats my fault. fixed 🙂